org.sugarlabs.MazeActivity.json

```diff
@@ -0,0 +1,242 @@
+{
+    "app-id": "org.sugarlabs.MazeActivity",
```

Comment:
Rename it to `org.sugarlabs.Maze` (as you have it in your activity.info). Make sure to keep this consistent everywhere, including the .json file name.

Reply:
9656eaa